File: 14659.py
N = int(input())
peaks = list(map(int, input().split()))

cnt = 0
highest = 0
res = []

for i in peaks:
    if i < highest:
        cnt += 1
    else:
        highest = i
        res.append(cnt)
        cnt = 0

res.append(cnt)  #내림차순일때?
print(max(res))

File: 14659.js
//input.txt  /dev/stdin
const fs = require('fs');
let input = fs.readFileSync('input.txt').toString().trim().split('\n');
let N = input[0];
let mountin = input[1].split(' ').map(el => Number(el));

answer = [];
highest = 0;
cnt = 0;
for(let i of mountin){
    if(i < highest){
        cnt ++
    }else{
        highest = i
        answer.push(cnt);
        cnt = 0;
    }
}
answer.push(cnt)
console.log(Math.max(...answer))
